sycl/CMakeLists.txt

```diff
@@ -207,10 +211,16 @@ add_custom_target( sycl-toolchain
           llvm-spirv
           llvm-link
           llvm-objcopy
+          xpti
           sycl-post-link
   COMMENT "Building SYCL compiler toolchain..."
 )
 
+# Add dependencies for building debug version while using the Microsoft compiler
+if (MSVC)
+  add_dependencies( sycl-toolchain xptid)
```

Comment:
Could you please guard all operations with xpti if SYCL_ENABLE_XPTI_TRACING  is disabled?

Reply:
Done!